src/rational.rs

```diff
@@ -113,6 +130,14 @@ impl Rational {
     }
 
     /**
+     * Returns this Rational to the nearest Int
+     */
+    pub fn round(&self) -> Int {
+        let r = self + Rational::new(self.sign().into(), 2.into());
```

Comment:
This would turn `n/d` into `n±1/(d + 2)` instead of `(n/d) ± (1/2) = (2n ± 1)/d`